janitor/functions.py

```diff
@@ -1181,7 +1181,7 @@ def remove_columns(df: pd.DataFrame, columns: List):
 
 
 @pf.register_dataframe_method
-def change_type(df, column: str, dtype):
+def change_type(df, column: str, dtype, ignore_exception = False):
     """
     Changes the type of a column.
 
```

Comment:
Could you also add a single sentence describing the intent of the ignore_exception, in plain English?

One suggestion to get you started is this:

```
Exceptions that are raised can be ignored. For example, if one has a mixed
dtype column that has non-integer strings and integers, and you want to coerce
everything to integers, you can optionally ignore the non-integer strings and 
replace them with ``NaN``s or keep the original value.
```